NumWords.java

```diff
@@ -1,6 +1,15 @@
 // Prints a given number using a hundreds, tens, and units notation.
 public class NumWords {
 	public static void main(String args[]) {
-	    // Replace this comment with your code
+		// Replace this comment with your code
+		int num = Integer.parseInt(args[0]);
+		int h, t, u; // h=hundreds, t=tens, u=units
```

Comment:
Good comment
but - since the names are very short, just use them as variables - 
int hundreds, int tens, int units